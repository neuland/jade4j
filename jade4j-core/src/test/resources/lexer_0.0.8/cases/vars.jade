- var foo = 'bar'
- var list = [1,2,3]
a(class=list, id=foo)