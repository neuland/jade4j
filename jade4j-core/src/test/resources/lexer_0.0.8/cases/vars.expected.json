{"type":"code","line":1,"val":"var foo = 'bar'","escape":false,"buffer":false}
{"type":"newline","line":2}
{"type":"code","line":2,"val":"var list = [1,2,3]","escape":false,"buffer":false}
{"type":"newline","line":3}
{"type":"tag","line":3,"val":"a","selfClosing":false}
{"type":"attrs","line":3,"attrs":[{"name":"class","val":"list","escaped":true},{"name":"id","val":"foo","escaped":true}]}
{"type":"eos","line":3}